src/migrations/0023_postgresql_11_to_13.py

```diff
@@ -58,7 +58,8 @@ def run(self):
             "LC_ALL=C pg_dropcluster --stop 13 main || true"
         )  # We do not trigger an exception if the command fails because that probably means cluster 13 doesn't exists, which is fine because it's created during the pg_upgradecluster)
         time.sleep(3)
-        self.runcmd("LC_ALL=C pg_upgradecluster -m upgrade 11 main")
+        self.runcmd("LC_ALL=C pg_upgradecluster -m upgrade 11 main || true")
+        time.sleep(3)
```

Comment:
Hmmm the sleep sounds legit but the `|| true` sounds really like a big nope ... Like, if the cluster upgrade miserably fail, you really don't want to yolo remove the current cluster because you may be dropping the entire databased before they get migrated to the new cluster x_x

Reply:
that's what we know now o_O'
We are restoring our db backup...

Reply:
@zamentur @alexAubin any conclusion on this?

Reply:
I dunno, didn't understand the previous comment ... was ljf meaning you lost the DB because brute-forcing the cluster upgrade ?

Anyway it just doesn't sound like a super good idea to yolo-ignore any error from the cluster upgrade and then yolodrop the old cluster, that's just a recipe for disaster

Reply:
yes we was a bit tired when we have done that... So this fix is not the good one. The true fix is to detect postgresql plugin and install the same for 13 version

Reply:
Soooooo closing then, but feel free to reopen if you want to iterate on this branch